docs/config.json

```diff
@@ -19,20 +19,21 @@
       ]
     },
     {
-      "icon": "wand",
-      "title": "Preview",
-      "entries": [
-        { "title": "Upcoming Releases", "slug": "/preview/upcoming-releases/" }
-      ]
-    },
-    {
-      "icon": "book",
-      "title": "Infrastructure Guides",
+      "icon": "kubernetes",
+      "title": "Kubernetes Access",
       "entries": [
-        { "title": "AWS", "slug": "/aws-oss-guide/" },
-        { "title": "GCP", "slug": "/gcp-guide/" },
-        { "title": "IBM", "slug": "/ibm-cloud-guide/" },
-        { "title": "OpenSSH Guide", "slug": "/openssh-teleport/" }
+        { "title": "Getting Started", "slug": "/kubernetes-access/getting-started/" },
+        {
+          "title": "Guides",
+          "slug": "/kubernetes-access/guides/",
+          "entries": [
+            { "title": "Multiple Clusters", "slug": "/kubernetes-access/guides/multiple-clusters" },
```

Comment:
Add trailing slashes here and below (`/kubernetes-access/guides/multiple-clusters/`) or it will mess up 3rd level nav as we found out in https://github.com/gravitational/next/issues/123.